pipelines/salesforce/__init__.py

```diff
@@ -0,0 +1,247 @@
+import typing as t
+from datetime import datetime
+
+import dlt
+from simple_salesforce import Salesforce
+
+
+def is_production() -> bool:
+    """Return True if the current environment is production.
+
+    You can override this function based on your own setup. It will
+    add a LIMIT 100 clause to the queries.
+    """
+    return True
+
+
+def load_contract(name: str) -> t.List[str]:
```

Comment:
We can drop this function, it was just something I used internally that I thought was a neat idea. If we want to reduce lines of code to the minimum working pipeline.

Reply:
Hi folks,  I tried running this pipeline on my end and am not able run it, after trying several times.
1. Credentials are passed using @configspec decorator, which asks you to declare a subdomain every time you run the pipeline but there is no provision in secrets.toml to enter the subdomain. And if we add it manually it again gives an error. 

2. There is domain in there. 

3. When you define that it gives you error again

4. When I tried to authenticate using a separate script without passing around credentials, then I got error 400 multiple times. 

5. Maybe If someone can help out with this it would be great. 

@adrianbr